code/game/objects/items/devices/flashlight.dm

```diff
@@ -165,13 +169,14 @@
 	name = "flare"
 	desc = "A red standard-issue flare. There are instructions on the side reading 'pull cord, make light'."
 	w_class = 2.0
-	brightness_on = 8 // Pretty bright.
-	light_power = 3
+	brightness_on = 4 // Pretty bright.
+	light_power = 4

```

Comment:
Light power should generally be 1 or under, can you post a screenshot of what this looks like?